playground/nuxt.config.ts

```diff
@@ -3,6 +3,7 @@ export default defineNuxtConfig({
     '@nuxt/ui',
     '../src/module'
   ],
+  // @ts-expect-error - Ignore invalid property error; mdc is a local module
```

Comment:
I feel this shouldn’t be needed. I think playground should be excluded in root tsconfig and playground type checking should be run separately.

Reply:
@danielroe updated 👍🏼